content/blog/2025/12/2025-12-12-twim.md

```diff
@@ -0,0 +1,196 @@
+>     - Superseded by [MSC4345: Server key identity and room membership](https://github.com/matrix-org/matrix-spec-proposals/pull/4345).
+> - [MSC3953: Server capability DAG](https://github.com/matrix-org/matrix-spec-proposals/pull/3953)
+>     - Superseded by [MSC4345: Server key identity and room membership](https://github.com/matrix-org/matrix-spec-proposals/pull/4345).
+> - [MSC4294: Ignore and mass ignore invites](https://github.com/matrix-org/matrix-spec-proposals/pull/4294)
+>     - Superseded by [MSC4155: Invite Filtering](https://github.com/matrix-org/matrix-spec-proposals/pull/4155)
+> 
+> ### Spec Updates
+> 
+> The Spec Core Team expects to release Matrix v1.17 next week with a few MSCs, along with some improvements to the spec website. Stay tuned for those!
+> 
+> Otherwise we're still working away at Matrix 2.0 MSCs - trying to keep them at the forefront while occasionally dipping into other MSCs. [MSC4143: MatrixRTC](https://github.com/matrix-org/matrix-spec-proposals/pull/4143), [MSC4354: Sticky Events](https://github.com/matrix-org/matrix-spec-proposals/pull/4354) and [MSC4186: Simplified Sliding Sync](https://github.com/matrix-org/matrix-spec-proposals/pull/4186) are the three currently being most actively worked on. The team appreciates your patience while the details are being hammered out.
+
+## Dept of Servers 🏢
+  
+### Synapse ([website](https://github.com/element-hq/synapse/))
+
+Synapse is a Matrix homeserver implementation developed by Element.
+
+[Devon Dmytro](https://matrix.to/#/@devonh:one.ems.host) announces
+
+> This week we released Synapse v1.144.0.
+> 
+> This release adds the following new features:
+> - Add experimental implementation of [MSC4380](https://github.com/matrix-org/matrix-spec-proposals/pull/4380) (invite blocking). ([\#19203](https://github.com/element-hq/synapse/issues/19203))
+> - Delayed event timeouts can now be restarted from workers. [#19207](https://github.com/element-hq/synapse/issues/19207)
+> 
+> ... and a whole lot more!
+> 
+> Thank you to all our contributors for helping to make Synapse the best it can be.
+> As always, feel free to stop by [#synapse:matrix.org](https://matrix.to/#/#synapse:matrix.org) to join in on the discussion and if you encounter a bug make sure to report it [here](https://github.com/element-hq/synapse/issues/new/choose).
+
+## Homeserver Deployment 📥️
+  
+### Element Server Suite ([website](https://element.io/server-suite))
+
+Element Server Suite (ESS) is a backend hosting solution for Matrix-based communications that supports self-hosted and fully managed deployments.
+
+[Gaël G.](https://matrix.to/#/@gaelg:element.io) says
+
+> ### [ESS Community 25.12.1](https://github.com/element-hq/ess-helm/releases/tag/25.12.1) and [ESS Pro 25.12.1](https://docs.element.io/latest/element-server-suite-pro/release-notes/) have been released!
```

Comment:
this isn't properly nested, and I discourage "misusing" headings like this as it ends up in the TOC this way:
<img width="283" height="145" alt="image" src="https://github.com/user-attachments/assets/b14d6759-cdb5-4305-8a41-a39e735e36be" />

```suggestion
> **[ESS Community 25.12.1](https://github.com/element-hq/ess-helm/releases/tag/25.12.1) and [ESS Pro 25.12.1](https://docs.element.io/latest/element-server-suite-pro/release-notes/) have been released!**
```